test/smoke/src/sql/areas/notebook/notebook.test.ts

```diff
@@ -55,5 +56,21 @@ export function setup() {
 			await app.workbench.sqlNotebook.waitForTrustedIcon();
 			await app.workbench.quickaccess.runCommand('workbench.action.closeActiveEditor');
 		});
+
+		it('No search results if search query is empty', async function () {
+			const app = this.app as Application;
+			await app.workbench.quickaccess.runCommand('Notebooks: Focus on Search Results View');
+			const results = await app.workbench.sqlNotebook.view.searchInNotebook('');
+			assert(results.children !== undefined && results.children.length === 0);
+		});
+
+		it('Simple query search works correctly', async function () {
+			const app = this.app as Application;
+			// Adding a regex expression to not depend on specific results of files
+			const regexExpr = /[0-9]+( results in )[0-9]+( files - )/;
+			await app.workbench.quickaccess.runCommand('Notebooks: Focus on Search Results View');
+			const results = await app.workbench.sqlNotebook.view.searchInNotebook('hello');
+			assert(results.textContent !== '' && (results.textContent.match(regexExpr) || results.textContent.includes('No results found')));
```

Comment:
Why is "No results found" a valid result for this? Aren't you supposed to have found something?